tck/src/main/java/org/eclipse/microprofile/lra/tck/TckInvalidParticipantSignaturesTests.java

```diff
@@ -120,7 +119,6 @@ public void invalidArgumentTypeInParticipantMethodTest() {
     private void testInvalidDeployment(String deploymentName) {
         deploymentNameRule.deploymentName = deploymentName;
         expectedException.expect(DeploymentException.class);
-        expectedException.expectMessage(InvalidLRAParticipantDefinitionException.class.getSimpleName());
```

Comment:
I suppose you should verify that the exception was a RTE as that would match the wording on line 47

Reply:
But if the implementation throws any subclass of RuntimeException the text "RuntimeException" won't be present in the stacktrace. This is why I removed the check.

Reply:
or @tomjenkinson I will rewrite the issue and PR to include your suggestion about only failing the deployment (thus not mentioning RuntimeException) Thanks for looking into this!

Reply:
No problem - I would be happy with either approach - as you say a subclass of RTE might need some code to do with instanceof and construct it by classname (and expect the Jar in the classpath and so on) only perhaps to be removed if we agree to go for the wording to just fail the deployment :)